### DISCUSSION TITLE
Getting started with Pywr and ChatGPT

### DISCUSSION THREAD

**Discussion by edpotter6** [pywr/pywr#1085] [General]
Hello Pywr community,
I firstly wanted to ask if there have been some updates on the Pywr code since 2021 such that the commands 
model = pywr.SystemModel() and model = pywr.Model() no longer work to create a new model?

Could someone tell me what the current command is?

The reason I ask that with the advent of ChatGPT I have found coding much more accessible, however for this particular package it's use seems to be slightly limited (partly because ChatGPT is trained on data pre September 2021). I wonder as a suggestion to the writers if it may be worth reverting some of the very basic commands in Pywr such as that to create a model, to what they were pre-2021 as this may enable novice coders like me to enter in to using the package much more quickly than they would otherwise?
If it's just a one off it may annoy more regular users than it does help new ones!

The other flaw (and there are several with ChatGPT) is that has pointed me to the following link for help https://pywr.github.io/pywr-docs/ (to try to find the new command!). Not sure if this link has ever existed, but wondering if it could be used to signpost traffic trying to learn Pywr via ChatGPT to where the latest documentation is! I am also reasonably new to GitHub so it's not all straight forward.

**Comment by jetuk** [comment]
Hi,

I am not sure ChapGPT is going to work too well with Pywr specifically. I understand it can help with general Python programming, but I would guess Pywr is too specific. For example, I don't think `SystemModel` has ever been a class in Pywr, and I don't think it has ever worked to do `pywr.Model()` (but perhaps it should). I think it's just making up nonsense here. 

You need to import the `Model` class from the `pywr.model` module, like so:

```python
from pywr.model import Model

model = Model()
```

The documentation is [here](https://pywr.github.io/pywr/). Which is the link at the top of the README file in this repository. It looks like ChatGPT has found an old documentation repo that is no longer used.
